cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDatabaseBase.java

```diff
@@ -287,7 +287,6 @@ public Response remove(String id, String rev) {
                     DeserializationTypes.LC_RESPONSES);
             for(Response response : bulkResponses) {
                 response.setStatusCode(connection.getConnection().getResponseCode());
-                response.setReason(connection.getConnection().getResponseMessage());
```

Comment:
Think this method deserves a javadoc updating regarding the response code, just for clarity.

Reply:
Fixed in 3002eb0